lib/masamune/filesystem.rb

```diff
@@ -157,6 +157,22 @@ def touch!(*files)
       end
     end
 
+    def size(file)
+      case type(file)
+      when :hdfs
+        hadoop_fs('-test', '-z', file)
```

Comment:
@adithyar-vmw I'm not sure this will return the file size using `hadoop`:

```
$ echo 'file size test' > /tmp/test_file.txt
$ ls -la /tmp/test_file.txt
-rw-r--r--  1 mandrews  wheel  15 Jan  5 15:29 /tmp/test_file.txt

$ hadoop fs -test -z /tmp/test_file.txt
17/01/05 15:29:34 WARN util.NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
```

seems like we want something like this:
```
$ hadoop fs -ls /tmp/test_file.txt
17/01/05 15:32:00 WARN util.NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
-rw-r--r--   1 mandrews wheel         15 2017-01-05 15:29 /tmp/test_file.txt
```